exporter/exporterhelper/tracehelper.go

```diff
@@ -26,25 +26,33 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector/observability"
 )
 
-// PushTraceData is a helper function that is similar to ConsumeTraceData but also returns
-// the number of dropped spans.
-type PushTraceData func(ctx context.Context, td consumerdata.TraceData) (droppedSpans int, err error)
+// Suffix to use for span names emitted by exporter for observability purposes.
+const spanNameSuffix = ".ExportTraceData"
 
+// traceDataPusher is a helper function that is similar to ConsumeTraceData but also
+// returns the number of dropped spans.
```

Comment:
we can do this later but it is more precise to say "number of spans not sent", however, unless if we are talking about translation issues it should be always the size of the td.Spans

Reply:
Let's merge as is, we use "dropped spans" everywhere in the code base that uses this functionality.